code/__HELPERS/ai.dm

```diff
@@ -0,0 +1,34 @@
+//Returns a node that is in the direction of this node; must be in the src's adjacent node list
+/obj/effect/ai_node/proc/GetNodeInDirInAdj(dir)
+
+	if(!datumnode.adjacent_nodes || !datumnode.adjacent_nodes.len)
```

Comment:
```suggestion
	if(length(!datumnode.adjacent_nodes))
```

Reply:
`if(!length(datumnode.adjacent_nodes))`
i think you mean @ninjanomnom

Reply:
yes

Reply:
Implemented suggestion